README.md

```diff
@@ -1,26 +1,16 @@
 # AI studio
 
-## Installing a released version
+The Podman Desktop AI Studio extension simplifies getting started and developing with AI in a local environment.  It provides key open-source technologies to start building on AI.  A curated catalog of so-called recipes helps navigate the jungle of AI use cases and AI models.  AI Studio further ships playgrounds: environments to test and refine AI models, for instance, a chat bot.
```

Comment:
Refining models is not part of the playground user experience, only playing and testing models

Reply:
I've seen mockups showing to parametrize the models in the playgrounds.  Are there still plans to do that?